projects/fsastorefrontlib/src/cms-components/b2b/units.config.ts

```diff
@@ -0,0 +1,211 @@
+              breadcrumb: 'orgUnit.breadcrumbs.list',
+            },
+          },
+        },
+        children: [
+          {
+            path: 'create',
+            component: UnitFormComponent,
+          },
+          {
+            path: `:${ROUTE_PARAMS.unitCode}`,
+            component: FSUnitDetailsComponent,
+            data: {
+              cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.details' },
+            },
+            children: [
+              {
+                path: 'edit',
+                component: UnitFormComponent,
+              },
+              {
+                path: 'children',
+                component: UnitChildrenComponent,
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.children' },
+                },
+                children: [
+                  {
+                    path: 'create',
+                    component: UnitChildCreateComponent,
+                  },
+                ],
+              },
+              {
+                path: 'approvers',
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.approvers' },
+                },
+                children: [
+                  {
+                    path: '',
+                    component: UnitAssignedApproverListComponent,
+                  },
+                  {
+                    path: 'assign',
+                    component: UnitApproverListComponent,
+                  },
+                ],
+              },
+              {
+                path: 'users',
+                component: UnitUserListComponent,
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.users' },
+                },
+                children: [
+                  {
+                    path: 'create',
+                    component: UnitUserCreateComponent,
+                  },
+                  {
+                    path: `:${ROUTE_PARAMS.userCode}/roles`,
+                    component: UnitUserRolesFormComponent,
+                  },
+                ],
+              },
+              {
+                path: 'unitProductAssignments',
+                data: {
+                  cxPageMeta: {
+                    breadcrumb: 'orgUnit.breadcrumbs.productAssignment',
+                  },
+                },
+                children: [
+                  {
+                    path: '',
+                    component: AssignmentsComponent,
+                    children: [
+                      {
+                        path: 'assign',
+                        component: PotentialAssignmentsComponent,
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      guards: [AuthGuard, AdminGuard],
+    },
+  },
+};
+
+export const unitsTableConfigFactory: TableConfig = {
+  table: {
+    [OrganizationTableType.COST_CENTER]: {
```

Comment:
Lets put a TODO comment here with clarification why are product assignments component mapped to these OrganizationTableTypes. We should contact Spartacus team about that also.